samples/nrf9160/pdn/src/esm.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stddef.h>
+#include <zephyr.h>
+
+static const struct {
+	int reason;
+	const char *str;
+} esm_table[] = {
```

Comment:
This table is _really_ good to have, and is very helpful for developers. Small nitpicking from my side, could you add a comment on where you obtained the list?

Reply:
I'll try to dig it up, if possible.

Reply:
Added